cloud/amazon/kinesis.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = """
+---
+module: kinesis
+short_description: Creates an AWS Kinesis stream
+description:
+  - Creates an AWS Kinesis stream
+version_added: "2.0.0"
+author: "Sarah Haskins (@sarahhaskins)"
+options:
+  name:
+    description:
+      - Name of the AWS Kinesis stream to be created
+    required: true
+  shard_count:
+    description:
+      - Number of shards
+    required: true
+extends_documentation_fragment: aws
+"""
```

Comment:
should add a requirements field for boto3, currently the aws doc fragment only outputs boto